_posts/2024-10-02-ag3-cohorts-v20240924.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Ag3 cohorts analysis version 20240924"
+tags: data
+---
+
+A new cohorts analysis version `20240924` has been released for the
+Ag3 data resource. This is now the default cohorts analysis version
+when using the `malariagen_data` [Ag3
+API](https://malariagen.github.io/malariagen-data-python/latest/Ag3.html). This
+cohorts analysis is available for datasets up to and including Ag3.11.
```

Comment:
I have been following the approach where we declare up to the release that will be covered by this cohorts analysis, even if the release is not public yet. So I'd suggest:
```suggestion
cohorts analysis will be available for datasets up to and including Ag3.13.
```